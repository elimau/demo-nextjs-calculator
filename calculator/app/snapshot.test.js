import { render } from '@testing-library/react'
import Page from './page'

it('Renders calculator unchanged', () => {
  const { container } = render(<Page />)
  expect(container).toMatchSnapshot()
})
