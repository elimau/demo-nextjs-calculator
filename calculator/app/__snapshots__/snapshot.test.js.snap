// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders calculator unchanged 1`] = `
<div>
  <div>
    <h1>
      Home
    </h1>
    <a
      href="/about"
    >
      About
    </a>
  </div>
</div>
`;
